src/signals/incident/definitions/wizard-step-2-vulaan/afval.ts

```diff
@@ -38,6 +38,11 @@ export const controls = {
       label: 'Kies de container waar het om gaat',
       shortLabel: 'Container(s)',
       pathMerge: 'extra_properties',
+      wfsFilters: [
```

Comment:
I would use a string here instead of array. In this way the filter can be flexible/complex and support both `And` an `Or` operators.